HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+# CMSSW version specific customizations
+import os
+cmsswVersion = os.environ['CMSSW_VERSION']
+
+if cmsswVersion >= "CMSSW_7_5":
+
```

Comment:
Since this file is release dependent, there should be no need for a check on release.